app/views/users/_form.html.erb

```diff
@@ -0,0 +1,50 @@
+<%= form_for :user do |form| %>
+  
+
+  <div class="form-group">
+    <%= form.label :full_name %>
+    <%= form.text_field :full_name, :value => @full_name, class: "form-control" %>
+  </div>
+
+  <div class="form-group">
+    <%= form.label :email %>
+    <%= form.text_field :email, class: "form-control" %>
+  </div>
+
+  <div class="form-group">
+    <%= form.label :street_address %>
+    <%= form.text_field :street_address, class: "form-control" %>
+  </div>
+
+  <div class="form-group">
+    <%= form.label :street_address_second %>
+    <%= form.text_field :street_address_second , class: "form-control" %>
+  </div>
+
+  <div class="form-group">
+    <%= form.label :city %>
+    <%= form.text_field :city, class: "form-control" %>
+  </div>
+
+  <div class="form-group">
+    <%= form.label :state %>
+    <%= form.text_field :state, class: "form-control" %>
+  </div>
+
+  <div class="form-group">
+    <%= form.label :postal_code %>
+    <%= form.text_field :postal_code, class: "form-control" %>
+  </div>
+  <div class="form-group">
+    <%= form.label :phone_number %>
+    <%= form.text_field :phone_number, class: "form-control" %>
+  </div>
+
+  <select name="role[id]">
```

Comment:
You also have rails view helper called [select](https://apidock.com/rails/ActionView/Helpers/FormOptionsHelper/select) that can help you here